draft-ietf-quic-transport.md

```diff
@@ -1709,10 +1740,12 @@ controller.
 ### Response to a RST_STREAM
 
 Since streams are bidirectional, a sender of a RST_STREAM needs to know how many
-bytes the peer has sent on the stream.  If an endpoint receives a RST_STREAM
-frame and has sent neither a FIN nor a RST_STREAM, it MUST send a RST_STREAM in
-response, bearing the offset of the last byte sent on this stream as the final
-offset.
+bytes the peer has sent on the stream.   An endpoint that receives a RST_STREAM
+frame (and which has not sent a FIN or a RST_STREAM) MUST immediately respond
+with a RST_STREAM frame bearing the offset of the last byte sent on this stream
+as the final offset, and MUST NOT send any more data on the stream.  This
+RST_STREAM SHOULD contain the error code QUIC_RECEIVED_RST.
```

Comment:
So this seems wrong if the intent is to address the one-way stream use case.  Given that you have REQUEST_RST, that should be enough to cause the other side to abort.  Then your only obligation should be to end your side of the stream on your own time frame.  If that means that a hard abort needs both RST_STREAM and REQUEST_RST, I think that's OK.